test/scores-sfi.ts

```diff
@@ -0,0 +1,113 @@
+const { JsonRpcProvider } = require('@ethersproject/providers');
+const snapshot = require('../').default;
+const networks = require('@snapshot-labs/snapshot.js/src/networks.json');
+
+const space = 'turpintinz.eth';
+const network = '1';
+const snapshotBlockNumber = 13623707;
+const strategies = [
+  {
+    "name": "saffron-finance-v2",
+    "params": {
+      "symbol": "SAFF_STAK_V2",
+      "decimals": 18,
+      "multiplier": 1.1,
+      "stakingPool": "0x4eB4C5911e931667fE1647428F38401aB1661763",
+      "singleAssets": [ "0xb753428af26E81097e7fD17f40c88aaA3E04902c", "0xf34960d9d60be18cc1d5afc1a6f012a723a28811" ]
+    }
+  },
```

Comment:
@turpintinz Why do we need this file? 🤔 we can just pass different params from `examples.json` and run `npm test --strategy=saffron-finance-v2`

Reply:
Nope, it can go. It was for our convenience. I'll make the change including changes to the README.